frontend/src/bundles/auth/components/SetUpForm.vue

```diff
@@ -119,9 +119,6 @@ export default {
     };
   },
 
```

Comment:
Add custom error if confirmPassword field is empty

Reply:
completed